flink-table/flink-table-common/src/test/java/org/apache/flink/table/types/extraction/TypeInferenceExtractorTest.java

```diff
@@ -427,8 +428,7 @@ private static Stream<TestSpec> functionSpecs() {
                                 "Could not find a publicly accessible method named 'eval'."),
 
                 // named arguments with overloaded function
-                TestSpec.forScalarFunction(NamedArgumentsScalarFunction.class)
-                        .expectNamedArguments("n"),
+                TestSpec.forScalarFunction(NamedArgumentsScalarFunction.class),
```

Comment:
What about add `expectxxx` blocks here?

Reply:
I have added relevant comments. We expect namedArgument to be null, so there is no expectXXX here.